README.md

```diff
@@ -173,7 +173,24 @@ The manual process of creating a new version is to:
     }
     ```
 
-### Handling breaking changes
+## Handling breaking changes
 Whenever a Breaking Change is introduced, the following steps should be taken in Go package:
 
 1. Edit `go.mod` file, and increase the version package suffix in the module name. For example, if the current version is `v2.0.0`, and you are releasing `v3.0.0`, the module name should be `github.com/asyncapi/spec-json-schemas/v3`.
+
+## SchemaStore compatibility testing
+
+AsyncAPI JSON Schema is referenced in [SchemaStore](https://www.schemastore.org/json/). In many IDEs, like VSCode, some extensions integrate with SchemaStore, like [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml). This way we enable autocompletion, validation and tooltips that helps writing AsyncAPI documents.
+
+Whenever you make changes in AsyncAPI JSON Schema, you should always manually verify that the schema is still supported by [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) and that it will be able to fetch and dereference it.
+
+- Make sure you have VSCode and YAML extension
+- Once you make changes in schemas, bundle them with `npm run bundle`
+- Edit [this sample AsyncAPI document](./test/fixtures/asyncapi.yml). Add a new field, hover over some property to see if tooltip still shows up. This sample document contains the following line that assures YAML uses your local schema and not the one from SchemaStore. Make sure it points to the schema bundle that you generated and that contains your changes:
+   ```yaml
+   # yaml-language-server: $schema=/Users/wookiee/sources/asyncapi-node/schemas/2.6.0-without-$id.json
```

Comment:
"wookiee"?  :-)

Reply:
hehe, yeah, that's what my computer is called, what I can do, I'm a fan of star wars and Chewbacca is my favourite 😆